index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css">
+  <title>Yuriy's Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <span class="logo">My Logo</span>
+      <nav>
+        <i class="fas fa-bars hidden"></i>
+        <ul class="menu">
+          <li class="menu-item">Portfolio</li>
+          <li class="menu-item">About</li>
+          <li class="menu-item">Contact</li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline">
+      <div class="main">
+        <h1 class="title">I'm Yuriy<br>Glad to see you!</h1>
+        <p class="main-text">I’m a software developer! I can help you build a product, feature or website. Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <span class="connect-text">LET'S CONNECT</span>
+        <div>
+          <ul class="icon-list">
+            <li><i class="fab fa-twitter"></i></li>
+            <li><i class="fab fa-linkedin-in"></i></li>
+            <li><i class="fab fa-github"></i></li>
+            <li><i class="fab fa-medium-m"></i></li>
+            <li><i class="fab fa-angellist"></i></li>
```

Comment:
- [ ] [OPTIONAL] I think it's good to make the media Icons clickable links and link them to your accounts this way visitors can easily visit them.